doc/source/notebooks/advanced/fast_predictions.pct.py

```diff
@@ -0,0 +1,108 @@
+# ---
+# jupyter:
```

Comment:
Still need to link this document from the notebook table of contents

Reply:
Good point - I have added this to the `intro.md` file.